libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -91,6 +85,15 @@ const AP_Param::GroupInfo AP_Camera::var_info[] = {
     // @Values: 0:TriggerLow,1:TriggerHigh
     // @User: Standard
     AP_GROUPINFO("FEEDBACK_POL",  9, AP_Camera, _feedback_polarity, 1),
+
+    // @Param: TRIGG_TIME
+    // @DisplayName: Duration that shutter is held open
+    // @Description: How long the shutter will be held open in seconds
+    // @Units: seconds
+    // @Increment: 0.01
```

Comment:
the increment is used for the number scroll boxes. I don't think the scroll box should use 0.000001. The increment isn't the smallest change that can be made, it's just the convenient change with the UI. Any number can be typed in